src/main/java/de/diddiz/LogBlock/listeners/BlockBreakLogging.java

```diff
@@ -30,16 +30,16 @@ public BlockBreakLogging(LogBlock lb) {
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onBlockBreak(BlockBreakEvent event) {
         if (isLogging(event.getBlock().getWorld(), Logging.BLOCKBREAK)) {
-            WorldConfig wcfg = getWorldConfig(event.getBlock().getWorld());
-            if (wcfg == null) {
+            WorldConfig worldConfig = getWorldConfig(event.getBlock().getWorld());
+            if (worldConfig == null) {

```

Comment:
While wcfg is definitely not a perfect name, it is ok as a local variable of limited scope. Don't see a reason for this change.